vendor.mod

```diff
@@ -49,7 +49,7 @@ require (
 	github.com/klauspost/compress v1.15.12
 	github.com/miekg/dns v1.1.43
 	github.com/mistifyio/go-zfs v2.1.2-0.20190413222219-f784269be439+incompatible
-	github.com/moby/buildkit v0.10.6
+	github.com/moby/buildkit v0.10.7-0.20230206124303-b8fdb4b78da0
```

Comment:
@tonistiigi do you know if there were other issues for v0.10.x and/or if there were plans for tagging a new version?